frontend/packages/noobaa-storage-plugin/src/models.ts

```diff
@@ -0,0 +1,40 @@
+import { K8sKind } from '@console/internal/module/k8s';
+
+export const NooBaaSystemModel: K8sKind = {
+  label: 'NooBaa System',
+  labelPlural: 'NooBaa Systems',
+  apiVersion: 'v1beta1',
+  apiGroup: 'noobaa.rook.io',
+  plural: 'noobaasystems',
+  abbr: 'nb',
+  namespaced: true,
+  kind: 'NooBaaSystem',
+  id: 'noobaasystem',
+  crd: true,
+};
+
+export const NooBaaBackingStoreModel: K8sKind = {
+  label: 'Backing Store',
+  labelPlural: 'Backing Stores',
+  apiVersion: 'v1beta1',
+  apiGroup: 'noobaa.rook.io',
+  plural: 'noobaabackingstores',
+  abbr: 'nbstore',
```

Comment:
`abbr` should only be 3 or maybe 4 characters and all caps

Reply:
Can we remove the abbr and not include the field? What is the use of abbr in this flow? We are using it as a shortcut for kubectl...

Reply:
The property might have a confusing name, but it's used in console for the resource type badges by the resource names. It's not related to the kubectl abbreviation. More than 3 or 4 characters doesn't work well.

<img width="437" alt="Search · OKD 2019-06-20 09-07-54" src="https://user-images.githubusercontent.com/1167259/59851562-f471dd80-933a-11e9-8ce2-320d83a7a9cc.png">

Reply:
Right, makes sense now.
So we can change to NB for the 'NooBaa System', NBST for the 'Backing Store' and NBC for the 'Bucket Classe'
What do you think?

Reply:
I'd suggest `NB` for system, `NBS` for backing store and `NBC` for bucket class.